_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -23702,9 +23702,6 @@
 /turf/open/floor/plasteel/neutral,
 /area/quartermaster/qm)
 "aXD" = (
-/obj/machinery/computer/security/mining{
```

Comment:
this is a important machine for QM because it helps him manage minning

Reply:
he has one already, in his private office.

Reply:
there are 2 on Deltastation

Reply:
this is his office?